tests/phpunit/tests/rest-api/rest-block-type-controller.php

```diff
@@ -249,6 +250,9 @@ public function test_get_item_invalid() {
 		$this->assertNull( $data['category'] );
 		$this->assertNull( $data['textdomain'] );
 		$this->assertFalse( $data['is_dynamic'] );
+		// TODO: this is unexpected, but it looks like we're transforming scalars to arrays in rest_sanitize_value_from_schema
+		// This actually returns array( array() ); Need to step through to see what's happening
```

Comment:
Highlighting some unexpected validation behavior. This returns `array( array() )` instead of `array()`. It looks like we're transforming scalars to arrays in `rest_sanitize_value_from_schema`

Reply:
That is happening because a string is allowed for an array value, it'll get casted to an array. Then the string is getting converted to an empty object ( which is an array ) when each array item is validated.

Reply:
Thanks, it looks like this might be the only example where we have an array of objects, which is why it stuck out to me in the test cases. The default might not be useful, but I updated the expectation 👍